packages/expo-updates/ios/EXUpdates.podspec

```diff
@@ -17,6 +17,10 @@ Pod::Spec.new do |s|
   s.dependency 'React-Core'
   s.dependency 'EXStructuredHeaders'
 
+  s.pod_target_xcconfig = {
+    'GCC_TREAT_INCOMPATIBLE_POINTER_TYPE_WARNINGS_AS_ERRORS' => 'YES'
```

Comment:
Since the intent is to apply this setting to the entire project, shouldn't this configuration go inside the app's Podfile?

Reply:
A few things:
- As far as I can tell cocoapods doesn't provide a way to configure this in a Podfile, only in podspec. The odd thing is that there is a way for a podspec to dictate flags for the resulting workspace I believe https://guides.cocoapods.org/syntax/podspec.html#tab_user_target_xcconfig but not a way for the user project to dictate that setting itself.
- We [`inhibit_all_warnings`](https://guides.cocoapods.org/syntax/podfile.html#tab_inhibit_all_warnings_bang) in the podfile so in each dependency of the app we wish to see this warning we need to change it to an error (this change)

> shouldn't this configuration go inside the app's Podfile?

Maybe I'm confused, but I don't believe the app has a Podfile?

Reply:
The app has a Podfile at expo/ios/Podfile (it needs a Podfile to specify deps). One thing that's confusing to me is that GCC_TREAT_INCOMPATIBLE_POINTER_TYPE_WARNINGS_AS_ERRORS in expo-updates affects RNViewShot -- it seems like these settings are leaking out of each pod.

Reply:
Ah, that's what the change in the .pbxproj is. It's the corresponding change for the app since the Podfile only generates the workspace and doesn't modify the original project's settings. This PR makes the same change in two different ways, one for a library that can be done in the podspec and the other manually in Xcode itself. Sidenote: it's annoying that GitHub hid the two line change to the pbxproj.

Reply:
I see. I thought the pbxproj changes were applied by `pod install` taking the config setting from expo-updates.
Could you make the same change to apps/bare-expo/ios?